core/src/toga/app.py

```diff
@@ -496,6 +498,11 @@ def __init__(
     def _create_impl(self):
         self.factory.App(interface=self)
 
+    @property
+    def screens(self) -> tuple[Screen, ...]:
```

Comment:
This type annotation is incorrect - it's returning a `list[Screen]`, not a tuple (and a list is what it *should* be returning - a tuple implies there is positional significance).

Reply:
Corrected it. Thanks.